src/Common/src/TypeSystem/Common/DefType.FieldLayout.cs

```diff
@@ -2,35 +2,81 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Diagnostics;
 
 namespace Internal.TypeSystem
 {
     // This is the api surface necessary to query the field layout of a type
-
     public abstract partial class DefType : TypeDesc
     {
+        private class FieldLayoutFlags
+        {
+            public const int HasContainsPointers = 1;
```

Comment:
I think making the change as part of this change would be appropriate. However, if you want to make it as a followon change, I'd review that as good too.

LGTM